modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/QuestionRow.es.js

```diff
@@ -60,74 +66,125 @@ export default function QuestionRow({
-									: 'caret-top'
-							}
-							tooltip={Liferay.Language.get('votes')}
-							value={normalizeRating(question.aggregateRating)}
-						/>
-					</li>
-
-					<li>
-						<QuestionBadge
-							symbol="view"
-							tooltip={Liferay.Language.get('view-count')}
-							value={question.viewCount}
-						/>
-					</li>
-
-					<li data-testid="has-valid-answer-badge">
-						<QuestionBadge
-							className={
-								question.hasValidAnswer
-									? 'alert-success border-0'
-									: ''
-							}
-							symbol={
-								question.hasValidAnswer
-									? 'check-circle-full'
-									: 'message'
-							}
-							tooltip={Liferay.Language.get('number-of-replies')}
-							value={question.numberOfMessageBoardMessages}
-						/>
-					</li>
-
-					{items && !!items.length && (
-						<li>
-							<ClayDropDownWithItems
-								className="c-py-1"
-								items={items}
-								trigger={
-									<ClayButtonWithIcon
-										displayType="unstyled"
-										small
-										symbol="ellipsis-v"
+			<div
+				className={classNames(
+					'align-items-center',
+					'mb-0',
+					'd-flex',
+					'flex-wrap',
+					{
+						'justify-content-between': display.styled,
+					}
+				)}
+			>
+				{display.linksLayer && (
+					<>
+						<span>
+							{showSectionLabel && (
+								<SectionLabel
+									section={question.messageBoardSection}
+								/>
+							)}
+						</span>
+						<ul className="c-mb-0 d-flex flex-wrap list-unstyled mb-3 stretched-link-layer">
+							<li>
+								<QuestionBadge
+									symbol={
+										normalizeRating(
+											question.aggregateRating
+										) < 0
+											? 'caret-bottom'
+											: 'caret-top'
+									}
+									tooltip={Liferay.Language.get('votes')}
+									value={normalizeRating(
+										question.aggregateRating
+									)}
+								/>
+							</li>
+
+							<li>
+								<QuestionBadge
+									symbol="view"
+									tooltip={Liferay.Language.get('view-count')}
+									value={question.viewCount}
+								/>
+							</li>
+
+							<li data-testid="has-valid-answer-badge">
+								<QuestionBadge
+									className={
+										question.hasValidAnswer
+											? 'alert-success border-0'
+											: ''
+									}
+									symbol={
+										question.hasValidAnswer
+											? 'check-circle-full'
+											: 'message'
+									}
+									tooltip={Liferay.Language.get(
+										'number-of-replies'
+									)}
+									value={
+										question.numberOfMessageBoardMessages
+									}
+								/>
+							</li>
+
+							{items && !!items.length && (
+								<li>
+									<ClayDropDownWithItems
+										className="c-py-1"
+										items={items}
+										trigger={
+											<ClayButtonWithIcon
+												displayType="unstyled"
+												small
+												symbol="ellipsis-v"
+											/>
+										}
 									/>
-								}
+								</li>
+							)}
+						</ul>
+					</>
+				)}
+
+				{!display.linksLayer && (
```

Comment:
It is "!" hinders the verification because it will always deny the current value of the validation. 

I believe you can find a better way to do this by changing the value of the prop.